charger/easee.go

```diff
@@ -368,18 +394,56 @@ var _ api.ChargePhases = (*Easee)(nil)
 
 // Phases1p3p implements the api.ChargePhases interface
 func (c *Easee) Phases1p3p(phases int) error {
-	if phases == 3 {
-		phases = 2
-	}
+	var err error
+	if c.circuit != 0 {
+		// circuit level
+		uri := fmt.Sprintf("%s/sites/%d/circuits/%d/settings", easee.API, c.site, c.circuit)
```

Comment:
Es gibt inzwischen eine expliziten Endpoint um die Current zu steuern
https://developer.easee.cloud/reference/post_api-sites-siteid-circuits-circuitid-dynamiccurrent

Vorteil ist hier, dass man eine TTL in min übergeben kann. Wenn evcc offline wäre stellt sich der Circuit nach der TTL wieder auf die Ausgangsleistung (max) zurück. Würde hier aber erstmal einen relativ großen Wert nehmen (1 Tag). Phasenumschaltung passiert dann ja doch nicht regelmäßig.